shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/config/datasource/pool/creator/impl/HikariDataSourcePoolCreatorTest.java

```diff
@@ -55,6 +56,10 @@ public void assertCreateDataSourceConfiguration() {
         assertThat(configuration.getProps().get("password"), is("root"));
         assertThat(configuration.getProps().get("maximumPoolSize"), is(10));
         assertThat(configuration.getProps().get("minimumIdle"), is(1));
+        Properties dataSourceProperties = (Properties) configuration.getProps().get("dataSourceProperties");
```

Comment:
I think it may better assert dataSourceProperties is not null, otherwise will throw NPE.